src/PrestaShopBundle/Entity/Repository/OrderInvoiceRepository.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
```

Comment:
I may forgot but every PHP classes need the Header

Reply:
I've configured my phpstorm now to add the headers automatically